docs/setup/install-with-binary.md

```diff
@@ -31,7 +31,7 @@ There're two ways to download `cloudcore` binary.
 
 - Download from github release.
 
-    Now KubeEdge github officially holds three arch releases: amd64, arm, arm64. Please download the right package according to your platform.
+    Now KubeEdge github officially holds three arch releases: amd64, arm, arm64. Please download the right package with the correct release binary version (e.g v1.12.0) according to your platform.
```

Comment:
v1.12.0 is too old for us and we community no longer maintain this version. We can use newer version like v1.22.0/v1.21.0 for example.